src/GitHub.UI/Converters/BranchNameConverter.cs

```diff
@@ -0,0 +1,44 @@
+﻿using GitHub.Models;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Data;
+
+namespace GitHub.UI.Converters
+{
+    public class BranchNameConverter : IMultiValueConverter
+    {
+
+        public object Convert(object[] values, Type targetType, object parameter, CultureInfo culture)
+        {
+            var branch = values[0] as IBranch;
+            var activeRepo = values[1] as IRepositoryModel;
```

Comment:
Before accessing values here directly, you should check to make sure the array actually contains at least two elements and return if you don't